### PR TITLE
SDIT-2186: ✨ Start to create project - remove example api call

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: .github/workflows/security_owasp.yml [hunk 1]
```diff
@@ -2,11 +2,11 @@ name: Security OWASP dependency check
...
-    - cron: "48 8 * * MON-FRI" # Every weekday
+    - cron: "51 6 * * MON-FRI" # Every weekday at 06:51 UTC
 jobs:
   security-kotlin-owasp-check:
     name: Kotlin security OWASP dependency check
-    uses: ministryofjustice/hmpps-github-actions/.github/workflows/security_owasp.yml@v0.7 # WORKFLOW_VERSION
+    uses: ministryofjustice/hmpps-github-actions/.github/workflows/security_owasp.yml@v1 # WORKFLOW_VERSION
     with:
-      channel_id: C06KY7GLF3L
+      channel_id: ${{ vars.SECURITY_ALERTS_SLACK_CHANNEL_ID || 'NO_SLACK' }}
```
Comment (petergphillips) [inline .github/workflows/security_owasp.yml:11]: I've added the env var already

File: applicationinsights.dev.json [hunk 2]
```diff
@@ -16,16 +16,51 @@
   "sampling": {
     "percentage": 100,
     "overrides": [
+      {
+        "telemetryType": "dependency",
+        "attributes": [
+          {
+            "key": "db.system",
+            "value": "postgresql",
```
Comment (petergphillips) [inline applicationinsights.dev.json:24]: left postgres in here for now, need to amend to sql server but have to get app running with that first

File: helm_deploy/hmpps-historical-prisoner-api/values.yaml [hunk 3]
```diff
@@ -14,24 +13,15 @@ generic-service:
...
 
-  # Pre-existing kubernetes secrets to load as environment variables in the deployment.
-  # namespace_secrets:
-  #   [name of kubernetes secret]:
-  #     [name of environment variable as seen by app]: [key of kubernetes secret to load]
-
   namespace_secrets:
     hmpps-historical-prisoner-api:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
-      # Example client registration secrets
```
Comment (petergphillips) [inline helm_deploy/hmpps-historical-prisoner-api/values.yaml:32]: won't be calling out to any other apps

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**